README.md

```diff
@@ -1 +1,152 @@
+<!-- TABLE OF CONTENTS -->
 
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ](#faq)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖Reviewing my project from the trial<a name="about-project"></a>
+ 
+
+## Learning objectives
+
+- I will apply best practices in HTML code.
+- I will use semantic HTML tags.
+- I will use CSS selectors correctly.
+- I will use CSS box model.
+- I will perform a code review for yourself on a past project.
+
+
+## Description
+
+ - In the previous lessons, I got an introduction to the code review process and I upgraded my HTML and CSS skills. That means that I am ready to perform a code review on my code!
+
+## 🛠 Built With <a name="built-with"></a>
+
+<details>
+  <summary>HTML</summary>
+  <ul>
+    <li><a href="#">HTML</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>CSS</summary>
+  <ul>
+    <li><a href="#">CSS</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **[html]**
+- **[css]**
+
+- **colored header**
+- **milestone**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
```

Comment:
- [ ] Live demo link means the live version of your website that can be accessed from anywhere in the world.

Kindly consider adding the link as it is professional, and the viewer can easily find your website's online version from Readme. You can use GitHub Pages for this. Here is a [FreecodeCamp Link](https://www.freecodecamp.org/news/publish-your-website-netlify-github/) for reference.